### PR TITLE
update simd operators for special dims

### PR DESCRIPTION
**Pull request by LHT129**
update sq8 simd operators for  special dims (not power of 2)

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
lgtm

File: src/simd/sq8_simd_test.cpp [hunk 1]
```diff
@@ -54,11 +54,11 @@ TEST_CASE("SQ8 SIMD Compute Codes", "[SQ8 SIMD]") {
     std::vector<uint8_t> vec1, vec2;
     for (const auto& dim : dims) {
         auto vec = fixtures::generate_vectors(count * 2, dim);
-        vec1.resize(count * dim * 2, 0);
+        vec1.resize(count * dim);
```
Comment (ShawnShawnYou) [inline src/simd/sq8_simd_test.cpp:57]: modify all self-defined dims to common_used_dims

Comment (LHT129) [reply src/simd/sq8_simd_test.cpp:57]: done

**Review by ShawnShawnYou — APPROVED** [review]

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**